libs/platform/src/lib/form/select/select/select.component.ts

```diff
@@ -101,6 +101,13 @@ export class SelectComponent extends BaseSelect implements AfterViewInit, AfterV
         // setting option items
         this.select._options.forEach((option) => {
             const optionItem = option._getHtmlElement();
+
+            // set title-text items to center - useful for two columns mainly
+            const listTitleText = optionItem.querySelector('.fd-list__title-text');
+            if (listTitleText) {
+                listTitleText.setAttribute('style', 'display:flex; align-items:center');
+            }
```

Comment:
I don't like this solution. We don't need to use `querySelector` and we can add this css to the component itself OR better fix it in the fund-styles